### DISCUSSION TITLE
Add a splash screen

### DISCUSSION THREAD

**Discussion by pree-T** [TeamAmaze/AmazeFileManager#3137] [Ideas]
A splash screen will give a professional touch to the app and enhance its visual appeal.

**Comment by VishnuSanal** [comment]
> A splash screen will give a professional touch to the app and enhance its visual appeal.

Does that follow our design? Google recommends to not use splash screen. Plus I think adding a splash screen all of a sudden would be bad. We are adding up to start up time unnecessarily. Right?

**Reply by EmmanuelMess** [reply]
> Google recommends to not use splash screen.

A reminder that google doesn't follow their own advice.

> We are adding up to start up time unnecessarily. Right?

No, it should use the load time of the MainActivity. It technically adds a few millis, but that doesn't matter IMO.

**Reply by VishnuSanal** [reply]
Oh, then I think we can maybe implement [this](https://developer.android.com/guide/topics/ui/splash-screen) for the future (i.e, for Android 12+).

**Comment by EmmanuelMess** [comment]
See #3139

**Comment by shuvashish76** [comment]
It's like opposite to the feature I requested #1988  🤷🏿‍♂️
(_just to mention here, [this](https://github.com/zhanghai/MaterialFiles/issues/570) is MF dev's reply for the same issue_)

**Reply by VishalNehra** [reply]
Splash screen and startup flashing can coexist. For eg. See solid file explorer they have a dark startup banner so it doesn't seem that it's flashing, plus is a banner. So both problems solved
